Dynatrace/dynatrace-operator
# Consider all namespaceselectors in the Pod webhook

## Description



Corresponding ticket: https://dt-rnd.atlassian.net/browse/K8S-9333

With this the newly added (or more presicely: distributed) namespaceselectors are considered when in the Pod webhook. the namespaces that are stored in the request are checked against the namespaceselector that can be set in the DynaKube. 

## How can this be tested?



Deploy the operator with e DynaKube with a namespaceselector. Create namespaces that match the namespaceselector but also some that do not match it. Inject into those namespaces and check if everything works accordingly
